api/relay_message.go

```diff
@@ -21,7 +22,7 @@ const (
 type RelayMessageRequest struct {
 	// Required. cb58 encoding of the source blockchain ID for the message
 	BlockchainID string `json:"blockchain-id"`
-	// Required. Hex encoding of the warp message ID
+	// Required. cb58 encoding of the warp message ID
 	MessageID string `json:"message-id"`
 	// Required. Integer representation of the block number that the message was sent in
 	BlockNum string `json:"block-num"`
```

Comment:
Is there a reason we don't use an int here?

Reply:
Nope, changed :)